openlibrary/coverstore/code.py

```diff
@@ -512,3 +522,87 @@ def POST(self, category):
                 return 'cover has been deleted successfully.'
         else:
             return 'no such id: %s' % id
+
+
+def overlay_covers_over_background():
+    """This method take as an input a list with the five books, from a list,
+    and put their cover in the correct spot in order to create a new image for social-card"""
+
+    five_seeds = create_list_preview("/people/openlibrary/lists/OL1L")
+    background = Image.open("/openlibrary/static/images/Twitter_Post_Background_Shelf_Color.png")
+
+    logo = Image.open("/openlibrary/static/images/Open_Library_logo.png")
+    recom_text = create_preview_recommendation_text("/people/openlibrary/lists/OL1L")
+    para = textwrap.wrap(recom_text, width=35)
+    text_size=(344,33)
+    logo_size=(168,41)
+    text_position=(340,406)
+    logo_position=(823,439)
+    W, H = background.size
+    image = []
+    for seed in five_seeds:
+        cover = seed.get_cover()
+
+        if  cover:
+            response = requests.get(f"https://covers.openlibrary.org/b/id/{cover.id}-M.jpg")
+            image_bytes = io.BytesIO(response.content)
+
+            img = Image.open(image_bytes)
```

Comment:
These lines might make a nice separate utility function:
```suggestion
def get_cover_image(cover) -> Image:
    response = requests.get(f"https://covers.openlibrary.org/b/id/{cover.id}-M.jpg")
    return Image.open(io.BytesIO(response.content))
```